src/styles/page-template/src/_template.njk

```diff
@@ -50,6 +50,9 @@
         {% endblock %}
         {% block bodyEnd %}{% endblock %}
         <script type="module" src="/scripts/bundle.js"></script>
+        {% if isPatternLib %}
+            <script type="module" src="/scripts/patternlib.js"></script>
```

Comment:
The `nomodule` version of the patternlib script should also be included or the resizer wont work in IE11